django/chat/templates/chat/chat_readonly.html

```diff
@@ -16,14 +16,16 @@
   <main id="chat-outer" class="{{ mode }}">
     <div id="chat-flexbox" class="d-flex justify-content-between mb-3">
       <div id="chat-container" class="flex-grow-1">
-        <div class="position-fixed end-0 pe-3 pt-1 chat-sidebar-toggle text-muted"
-             style="z-index:1"
-             id="right-sidebar-toggle">
-          <span class="d-xl-inline d-none fw-semibold">{% trans "Read-only" %}</span>
-          <i class="bi bi-file-lock2"
-             style="font-size: 1.5rem;
-                    position: relative;
-                    top: 3px"></i>
+        <div id="shared-chat-warning"
```

Comment:
Switched to a warning banner so more text and author email link can be included.